model_definition.md

```diff
@@ -265,11 +265,11 @@ We use informative priors for parameters that have been well characterized in th
 | Total genomes shed per infected individual | $\log_{10}[G] \sim \mathrm{Normal}(9, 2)$ | Watson et al 2023[^Watson]   |
 | Initial infections per capita $I_0$ | $I_0 \sim \mathrm{Beta}(1 + k i_\mathrm{est}, 1 + k (1-i_\mathrm{est}))$ | where $i_\mathrm{est}$ is the sum of the last 7 days of hospital admissions, divided by jurisdiction population, and divided by the prior mode for $p_\mathrm{hosp}$, and $k = 5$ is a parameter governing the informativeness ("certainty") of the Beta distribution |
 | Initial exponential growth rate | $r \sim \mathrm{Normal}(0, 0.01)$ | Chosen to assume flat dynamics prior to observations |
-| Infection feedback term | $\gamma \sim \mathrm{logNormal}(6.37, 0.4)$ | Weakly informative prior chosen to have a mode of 500 in natural scale, based on posterior estimates of peaks from prior seasons in a few jurisdictions |
+| Infection feedback term | $\gamma \sim \mathrm{logNormal}(4.50, 0.64)$ | Weakly informative prior chosen to have a mode of 90 in natural scale, based on posterior estimates of the infection feedback fit to the peak in the 2023-24 epidemic season in a few jurisdictions |
```

Comment:
1. Would be ideal to provide the exact number of jurisdictions if you have it.
2. "To the season" should perhaps be clarified if possible. IIRC you used a longer fitting window than the 90 day production window. Maybe just say the dates?

```suggestion
| Infection feedback term | $\gamma \sim \mathrm{logNormal}(4.50, 0.64)$ | Weakly informative prior chosen to have a mode of 90 in natural scale, based on posterior estimates of the infection feedback from model fits to the 2023-24 epidemic season in a few jurisdictions |
```

Reply:
Added the 5 states, it was multiple forecast dates (March 4th and March 11th) going back 365 days each.